react/features/base/participants/reducer.js

```diff
@@ -63,7 +64,8 @@ const DEFAULT_STATE = {
     remote: new Map(),
     sortedRemoteParticipants: new Map(),
     sortedRemoteScreenshares: new Map(),
-    speakersList: new Map()
+    speakersList: new Map(),
+    raisedHandsQueue: []
```

Comment:
It actually keeps the ids of the participants that have raised hands in the order that they raised their hands in. It is needed because we need the order to be able to display them in the list in that order